harmony/harmony.py

```diff
@@ -277,6 +284,8 @@ def __init__(self,
         self.skip_preview = skip_preview
         self.ignore_errors = ignore_errors
         self.grid = grid
+        self.capabilities = capabilities
```

Comment:
Request is a class for submitting transformation requests using harmony. I don't think we should try to also use it to submit calls to get the collection capabilities. Probably best to add a new simple class specifically for getting capabilities that just has two fields - a Collection object and an optional capabilities version.

Reply:
Sure. I will refactor the code to add a CapabilitiesRequest.